src/vcpkg/commands.install.cpp

```diff
@@ -234,11 +234,23 @@ namespace vcpkg
             // The VS2015 standard library requires comparison operators of T and U
             // to also support comparison of T and T, and of U and U, due to debug checks.
 #if _MSC_VER <= 1910
```

Comment:
We don't care about VS2015 so you can delete this if you want. Not worth resetting testing if that's the only change.